Our unicorn config requires dotenv

jbabe opened the pull request:
- In each project we have a unicorn config
  that does a Dotenv.load to reload any .env file
  changes. If the project using that config does not have the
  dotenv gem the unicorn:reexec command will not reload
  the .env
- setting to an older version 0.9 instead of 0.10 because
  our projects are using snapshot which only is compatible
  with dotenv (0.9.0)

Commit:
Our unicorn config requires dotenv

* In each project we have a unicorn config
that does a Dotenv.load to reload any .env file
changes. If the project using that config does not have the
dotenv gem the unicorn:reexec command will not reload
the .env
* setting to an older version 0.9 instead of 0.10 because
our projects are using snapshot which only is compatible
with dotenv (0.9.0)

ryanjones commented:
@mvandenbeuken what version of dotenv mucked with our stuff?

ryanjones commented:
That snapshot gem is a hacker.

mvandenbeuken commented:
0.9 tanks if there is a $ in an env variable value. I had to lock it in at 0.8 for GK.

https://github.com/amaabca/gatekeeper/commit/1749db2055a9b0146df8d0e8f6e99814c09f8548

jbabe commented:
https://github.com/bkeepers/dotenv/commit/cb6f4e678cc32c8bfb92f0192b72fac085c60dcf

ryanjones commented:
0.8 plz then :elephant: :elephant: :elephant:

Commit:
Fixing dotenv dependency at v0.0.8

* Shell variable expansion was added to
dotenv in v0.9.0 which causes some of
our .env files to break
* Rather than fixing all our .env files
just changing this for now (cause hopefully
we drop this gem)

ryanjones commented:
:elephant: :elephant: :elephant:
